plugin.video.hrmagplayer/resources/language/English/strings.po

```diff
@@ -0,0 +1,58 @@
+# Kodi Media Center language file

```

Comment:
please adopt the new resource folder layout for krypton:
`resources/language/resource.language.en_gb/strings.po`